mobxjs/mobx-state-tree
# chore: bump to v6.0.0 and build docs

## What does this PR do and why?

This PR will mark MST as v6. [Discussion here](https://github.com/mobxjs/mobx-state-tree/discussions/2178). In the last maintainer meeting we decided we'd move forward with this on Monday, May 6, 2024.

Before merging and shipping to npm, we'll want to run `bun run publish-docs` one more time to get everything published at the same commit.

## Steps to validate locally

`bun run test:all` should pass, and once we're ready with docs, we can re-run those, deploy, merge this, tag it, and publish to npm.
